tendrl/alerting/watcher/manager.py

```diff
@@ -0,0 +1,39 @@
+import etcd
+import logging
+import multiprocessing
+from tendrl.alerting.exceptions import AlertingError
+from tendrl.common.alert import Alert
+from tendrl.common.config import ConfigNotFound
+from tendrl.common.config import TendrlConfig
+
+config = TendrlConfig()
+LOG = logging.getLogger(__name__)
+
+
+class AlertsWatchManager(multiprocessing.Process):
```

Comment:
common's manager cannot be extended here because, it has certain things like the discovery_thread among others which are not used anywhere in monitoring as of now. Also this manager needs to be a multiprocessing.Process as it has a blocking etcd call in it and the common's manager being a greenlet will be blocked by a blocking call and hence the concurrency will be lost if used here.